src/ompl/tools/thunder/src/Thunder.cpp

```diff
@@ -70,6 +70,12 @@ void ompl::tools::Thunder::initialize()
 
 void ompl::tools::Thunder::setup()
 {
+    if (!filePath_.compare("unloaded"))
+    {
+        OMPL_WARN("Database was not set setup has to be called with a database different than %s", filePath_.c_str());
```

Comment:
Should be "Database filepath has not been set. Unable to setup.". You can also just use the `==` operator instead of compare.